en/docs/get-started/quick-start-guide.md

```diff
@@ -1,257 +1,477 @@
-# Quick Start Guide
+# Get Started with Choreo
```

Comment:
If we are calling this Get started, then let's have the file name with the same name too.. or change the title to match the file name.